src/openms/thirdparty/evergreen/src/BitReversedShuffle/BitReversal.hpp

```diff
@@ -0,0 +1,131 @@
+#ifndef _BITREVERSAL_HPP
+#define _BITREVERSAL_HPP
```

Comment:
@timosachsenberg @jpfeuffer if we include those into OpenMS headers, I think we need to put these into separate namespaces, e.g. `evergreen::` namespace. We dont want these in the global namespace...